docs/configuration/configuration.md

```diff
@@ -2131,6 +2168,11 @@ authorization:
   # It is mutually exclusive with `credentials`.
   [ credentials_file: <filename> ]
 
+# Optionally configures OAuth 2.0 authentication.
+# Cannot be used at the same time as basic_auth, or authorization.
```

Comment:
```suggestion
# Cannot be used at the same time as basic_auth or authorization.
```

Reply:
I would remove 'Optionally' (also from sigv4)

Reply:
maybe just `Optional OAuth2 configuration. Cannot be used at the same time as basic_auth or authorization.`